source/cpp/CMakeLists.txt

```diff
@@ -1,12 +1,4 @@
-add_library (
-  focusrite-e2e
-  include/focusrite/e2e/ClickableComponent.h
-  include/focusrite/e2e/Command.h
-  include/focusrite/e2e/CommandHandler.h
-  include/focusrite/e2e/ComponentSearch.h
-  include/focusrite/e2e/Event.h
-  include/focusrite/e2e/Response.h
-  include/focusrite/e2e/TestCentre.h
+add_library (focusrite-e2e STATIC
```

Comment:
I read somewhere (I'll try to dig it out) that it's best practice not to include `STATIC` if you don't know who is consuming your library. This means that the consumer can change libraries globally with `BUILD_SHARED_LIBS`.

I'm not sure where I read this, so I'm happy to include `STATIC` if there's a good reason.